package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "react-native-plaid-link-sdk",
-  "version": "6.0.3",
+  "version": "7.0.0-snapshot5",
   "description": "React Native Plaid Link SDK",
-  "main": "index.js",
-  "types": "/types/Types.d.ts",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "files": [
+    "build/**/*",
+    "android/**/*",
+    "ios/**/*",
+    "react-native-plaid-link-sdk.podspec"
+  ],
```

Comment:
i think a more widely used name for this type of folder would be `dist` instead of `build`.